modules/AOR_Reports/AOR_Report.php

```diff
@@ -1414,7 +1414,8 @@ public function build_report_query_join(
         if (!isset($query['join'][$alias])) {
             switch ($type) {
                 case 'custom':
-                    $customTable = $module->get_custom_table_name();
+                    $tmpModule = BeanFactory::getBean($module->module_dir);
```

Comment:
this make no difference, it's calling getBean in a bean that has already been passed in, so won't change the result from the get_custom_table_name and from my tests doesn't fix the issue

Reply:
@mattlorimer 
Sorry but it makes change.
In that case, $module->table_name has already been changed when the code reach that step.
add a log->fatal with $module->get_custom_table_name() and $tmpModule->get_custom_table_name() and you will see it.

Reply:
it may work in some scenarios, but it will create an intermittent issue where sometimes it will work and sometime it won't, as you have identified this requires a fix further up the chain.